### PR TITLE
Update installation_guide.rst

### PR DESCRIPTION
**Pull request by VoR0220**
update version requirements for node

### CODE REVIEW

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ Build from Source
 Linux
 =========
 
-Make sure you have `node.js <https://nodejs.org/en/>`_ and `npm <https://www.npmjs.com/>`_. If you don't, use your linux distro's package manager to get them.
+Make sure you have `.js <https://nodejs.org/en/>`_ and `npm <https://www.npmjs.com/>`_. If you don't, use your linux distro's package manager to get them.
```
Comment (LefterisJP) [inline docs/installation_guide.rst:118]: why delete node here?

Comment (VoR0220) [reply docs/installation_guide.rst:118]: must have been a mistake. Fixing.

**Comment by LefterisJP** [comment]
Hey @VoR0220 I hate to be nitpicky but can you squash your 2 commits in one? In general there should be 1 commit for 1 thing, and if you are just fixing something that the last commit had wrong you can just amend it.

If you don't know how to do it I can do it for you.

**Comment by VoR0220** [comment]
@LefterisJP haven't had to squash in a while. All set.

**Review by LefterisJP — APPROVED** [review]
LGTM. Thanks @VoR0220

**Pull request merged**